Docs  cards multi paredown (#1154)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/cards.mdx

</Column>
</Row>

Bottom-aligning the pictogram ensures that the card heading is the first element read. This helps when the message is not immediately decipherable from the pictogram.

An added bonus for the bottom-aligned pictogram is the feature of showing copy on hover. 

<Row>
<Column colMd={8} colLg={8}>

![Image of Card component](../../images/component/card/card-pictogram-motion.gif)

<Caption>Example of copy being shown on hover with bottom-aligned pictogram</Caption>

</Column>
</Row>

Top-aligning the pictogram should only be used when the pictogram is strongly and precisely associated with the card heading. For example, an airplane is the perfect choice for "Aerospace and defence", as shown in the above example. 

#### Static

While the entire default Card is clickable, the Static modifier makes it so only the text link is clickable on the card. This is useful for when you have multiple interactive elements on a single card such as CTAs and Tag links.

<Row>

<Column colMd={4} colLg={4}>

![Example image: single selection hover state of a parent tier filter item.](../../images/component/card/cards-modifier-card-static-outline.png)

> <Caption>Example of a Card with the static modifier and an outline</Caption>

Author: Agreed - I updated the language to hopefully be a little clearer. Also included a new gif animation to show how Card static can have multiple links to different destinations on one card.

### Content guidance for Pictogram card
When a pictogram is added to a Card, the Card's appearance and content guidance changes.

| Element                                                                 | Content type  | Required | Instances | Character count <br/>(English / translated) | Notes                   |
| ----------------------------------------------------------------------- | ------------- | -------- | --------- | ------------------------------------------- | ----------------------- |
| Heading                                                                 | Text          | Yes       | 1         | 40 / 55                                    |                         |
| Copy                                                                    | Text          | No        | 1         | 65 / 85                                    | Hover over copy on bottom-aligned pictogram only. |
| Pictogram                                                               | Element       | Yes       | 1         | –                                          | Top- or bottom-aligned |

## Card in card

> Card in card shares the same functionality and text content requirements of a standard card, but requires a large media. Card in card is often used in a [Card group](https://www.ibm.com/standards/carbon/components/card-group) to as a featured card.

Reviewer: ```suggestion
Card in card shares the same functionality and text content requirements of a standard card, but requires a large media element. Card in card is often used in a [Card group](https://www.ibm.com/standards/carbon/components/card-group) as a featured card.
```

<Row>
<Column colMd={8} colLg={8}>

![Image of Card in card component](../../images/component/card/card-options-card-in-card-anatomy.png)

</Column>
</Row>

1. **Media:** Can be an image or a video.
2. **Heading:** Heading has the highest information hierarchy on a card, and should be succinct and descriptive.

> 3. **Body copy:** Optional body copy can be added for additional descriptions.

Reviewer: ```suggestion
3. **Body copy:** Optional body copy can be added for additional description.
```

> 4. **CTA type:** When using an icon, by default will use `ArrowRight`.

Reviewer: ```suggestion
4. **CTA type:** When using an icon, the default is `ArrowRight`.
```

| Eyebrow                                                                 | Text          | No       | 1         | 20 / 32                                     |                         |
| Heading                                                                 | Text          | Yes      | 1         | 75 / 100                                    |                         |
| Media                                                                   | [Image](https://www.ibm.com/standards/carbon/components/images/) or [Video](https://www.ibm.com/standards/carbon/components/video)          | Yes       | 1         | –                                     | Full width, 16:9 aspect ratio                        |
| [CTA type](https://www.ibm.com/standards/carbon/components/cta)         | Option list   | Yes      | 1         | –                                           |                         |
| [CTA text](https://www.ibm.com/standards/carbon/components/cta)         | Link          | No       | 1         | –                                           | URL assigned in setup   |

## Card link

> Unlike Card, which can stand on its own, Card link should be presented with copy - it should never appear on its own. Card links contain a heading, an optional CTA (call-to-action), and optional body copy. The entire card is clickable.

Reviewer: ```suggestion
Unlike Card, which can be used as a standalone component, Card link should always be accompanied by contextual copy on the page. Card links contain a heading, an optional CTA (call-to-action), and optional body copy. The entire card is clickable.
```

<Row>
<Column colMd={8} colLg={8}>

![Image of Table of contents component](../../images/component/card/card-options-card-link-anatomy.png)

</Column>
</Row>

1. **Heading:** Heading has the highest information hierarchy on a card, and should be succinct and descriptive.

> 2. **Body copy:** Optional body copy can be added for additional descriptions.

Reviewer: ```suggestion
2. **Body copy:** Optional body copy can be added for additional description.
```

> 3. **CTA type:** When using an icon, by default will use `ArrowRight`.

Reviewer: ```suggestion
3. **CTA type:** When using an icon, the default is `ArrowRight`.
```

| Heading                                                                 | Text          | Yes       | 1         | 25 / 35                                    |                         |
| Copy                                                                    | Text          | No        | 1         | 40 / 55                                    |                         |
| [CTA type](https://www.ibm.com/standards/carbon/components/cta)         | Option list   | Yes       | 1         | -                                          |                         |

## Feature card

Feature card is used for highlighting a piece of content on page. It should be used sparingly to be effective. Feature card has two sizes: medium and large. 

### Modifiers

#### Size
Medium is the default size, occupying 8 columns and consisting of an image, heading, and CTA (call-to-action).

<Row>
<Column colMd={8} colLg={8}>

![Image of Feature card - medium](../../images/component/card/card-options-feature-card-medium-anatomy.png)

</Column>
</Row>

1. **Image:** Must use a 1:1 aspect ratio.
2. **Heading:** Heading has the highest information hierarchy on a card, and should be succinct and descriptive.

> 3. **CTA type:** When using an icon, by default will use `ArrowRight`.

Reviewer: ```suggestion
3. **CTA type:** When using an icon, the default is `ArrowRight`.
```

4. **Type:** Determines the size of the Feature card.

> The large feature card occupies 12 columns and allows for the addition of an eyebrow as well as body copy. 

Reviewer: ```suggestion
The large Feature card occupies 12 columns and allows for the addition of an eyebrow as well as body copy. 
```

<Row>
<Column colMd={8} colLg={8}>

![Image of Feature card - large](../../images/component/card/card-options-feature-card-large-anatomy.png)

</Column>
</Row>

1. **Image:** Must use a 1:1 aspect ratio.
2. **Eyebrow:** Add additional context to the the heading.
3. **Heading:** Heading has the highest information hierarchy on a card, and should be succinct and descriptive.
4. **Body copy:** Optional body copy can be added for additional descriptions.

> 5. **CTA type:** When using an icon, by default will use `ArrowRight`.

Reviewer: ```suggestion
5. **CTA type:** When using an icon, the default is `ArrowRight`.
```
